### PR TITLE
Throttle change events from Scaled Slider

### PR DESCRIPTION
**Pull request by mfurseman**
Use a timer to prevent rapid caputs from occurring when the slider value is dragged.

### CODE REVIEW

**Comment by mfurseman** [comment]
@willrogers @nickbattam

File: applications/opibuilder/opibuilder-plugins/org.csstudio.swt.widgets/src/org/csstudio/swt/widgets/figures/ScaledSliderFigure.java [hunk 1]
```diff
@@ -449,8 +452,19 @@ public void mouseDragged(MouseEvent me) {
...
+                        if(value != oldValue) {
+                            if(timer == null) {
+                                timer = new OPITimer();
+                            }
+                            if(timer.isDue()) {
+                                timer.start(new Runnable() {
+                                    @Override
+                                    public void run() {
+                                        fireManualValueChange(value);
+                                        System.out.println("Timer called");
```
Comment (nickbattam) [inline applications/opibuilder/opibuilder-plugins/org.csstudio.swt.widgets/src/org/csstudio/swt/widgets/figures/ScaledSliderFigure.java]: debug print statement

**Comment by nickbattam** [comment]
This will ignore updates made while the timer is running. Does this lead to a situation where the final sliding update(s) are not passed to the PV?

File: applications/opibuilder/opibuilder-plugins/org.csstudio.swt.widgets/src/org/csstudio/swt/widgets/figures/ScaledSliderFigure.java (hunk 1 above)
Comment (mfurseman) [reply applications/opibuilder/opibuilder-plugins/org.csstudio.swt.widgets/src/org/csstudio/swt/widgets/figures/ScaledSliderFigure.java]: bummer

**Comment by mfurseman** [comment]
The timer reads the 'value' when it fires. so it takes whatever was the last updated value at the end of it's 100ms. It should write the final value to the PV.

**Comment by willrogers** [comment]
This looks like a clean solution to me. I updated the comment to help me understand - does it make sense to you two? If so, we should merge.

**Pull request merged**